doc/src/manual/unicode-input.md

```diff
@@ -0,0 +1,66 @@
+# Unicode Input
+
+!!! warning
+
+    This table may appear to contain missing characters in the second column, or even
+    show characters that are inconsistent with the characters as they are rendered in
+    the Julia REPL. In these cases, users are strongly advised to check their choice
+    of fonts in their browser and REPL environment, as there are known issues with
+    glyphs in many fonts.
+
+```@eval
+#
+# Generate a table containing all LaTeX and Emoji tab completions available in the REPL.
+#
+
+function tab_completions(symbols...)
+    completions = Dict{String, Vector{String}}()
+    for each in symbols, (k, v) in each
+        completions[v] = push!(get!(completions, v, String[]), k)
+    end
+    return completions
+end
+
+function unicode_data()
+    file = normpath(JULIA_HOME, "..", "..", "doc", "UnicodeData.txt")
+    url = "http://www.unicode.org/Public/UCD/latest/ucd/UnicodeData.txt"
```

Comment:
Yes, probably would be better to use a stable source for it. With this rewrite I just made sure it was doing the same thing as before rather than going and changing anything like that.

> Or couldn't we get this information from utf8proc itself?

The file is only used to lookup the unicode names for chars, so if utf8proc can do that then that would be great.